src/annotator/test/delegator-test.js

```diff
@@ -0,0 +1,48 @@
+import Delegator from '../delegator';
+
+describe('Delegator', () => {
+  it('constructor sets `element` and `options` properties', () => {
```

Comment:
Funky grammar on this description...

Does this test have meaning in the future? It feels like it validates that the current changeset "works" but can you expand on what it's testing at a higher level?

Reply:
> Funky grammar on this description...

Yeah, but it will still make sense if you see it in a test failure log.

> Does this test have meaning in the future?

It's just testing that the initial state of the object is what is expected. Quite what that initial state will be might change in the future.